README.md

```diff
@@ -0,0 +1,117 @@
+# Web Test Transformer
+
+- [Web Test Transformer](#web-test-transformer)
+  - [Overview](#overview)
+  - [Setup](#setup)
+  - [Usage](#usage)
+    - [Dry Run](#dry-run)
+    - [CLI Flags](#cli-flags)
+      - [Required Flags](#required-flags)
+  - [Verification](#verification)
+    - [All PASS Requirement](#all-pass-requirement)
+  - [Issues](#issues)
+
+## Overview
+Web Test Transformer is designed to transform HTML tests in Chromium source from the legacy
+[js-test.js](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/web_tests/resources/js-test.js)
+framework to the current
+[testharness.js](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/web_tests/resources/testharness.js)
+framework.
+
+
+View the design document at [go/web-test-transformer](go/web-test-transformer) (internal Google access only).
```

Comment:
I think somewhere near the top we should outline the steps the user should take, something like:
1. run the tool on some tests to transform them and verify they work in run_web_tests
2. manually review the transformation and any verification errors, and make adjustments where needed
3. commit any changes that you want to keep
4. mail a CL for review and cq run